client/packages/host/src/components/AuthenticationAlert.tsx

```diff
@@ -46,7 +45,8 @@ export const AuthenticationAlert = () => {
     }
 
     if (error === AuthError.PermissionDenied) {
-      LocalStorage.removeItem('/auth/error');
+      toggleOff();
+      setTimeout(removeError, 200);
```

Comment:
not ideal - without the debounce, the internal state doesn't have time to update and the auth alert is shown twice - once on the settings page and again on the dashboard 🙄